### PR TITLE
Add the beer team as CODEOWNERS

### PR DESCRIPTION
**Pull request by nicklewis**

### CODE REVIEW

**Comment by nicklewis** [comment]
@mwaggett @steveax

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+* @mwaggett @nicklewis @steveax
```
Comment (mwaggett) [inline CODEOWNERS]: can we create a Team for this?

Comment (nicklewis) [reply CODEOWNERS]: We can... Is it easier to manage a team than a CODEOWNERS file? Assuming that team will only be responsible for one repo

Comment (mwaggett) [reply CODEOWNERS]: 1. I think it *is* easier to manage a team than a file, and 2. who knows what other beer-related repos we may acquire 😜

**Comment by nicklewis** [comment]
Updated to use @puppetlabs/beer

**Review by mwaggett — APPROVED** [review]

**Review by steveax — APPROVED** [review]

**Pull request merged**